modules/gdscript/gdscript.h

```diff
@@ -163,13 +163,14 @@ class GDScript : public Script {
 	HashSet<PlaceHolderScriptInstance *> placeholders;
 	//void _update_placeholder(PlaceHolderScriptInstance *p_placeholder);
 	virtual void _placeholder_erased(PlaceHolderScriptInstance *p_placeholder) override;
+	void update_exports_down(bool p_base_exports_changed);
```

Comment:
```suggestion
	void _update_exports_down(bool p_base_exports_changed);
```
Let's keep the private method style consistent here

Reply:
Fixed in new commit.